### ISSUE TITLE
[ENHANCEMENT] Increase font visibility difference of user input and Claude's output

### ISSUE THREAD

**Issue by webmemo-code** [anthropics/claude-code#3114]
I suggest increasing the visibility difference of the font between what the user tells Claude and Claude's response.

When scrolling through long sessions, on Windows I find it hard to identify my input between Claude's output.

Attached please find an example. My user input is in light gray, Claude's answer looks very similar.

![Image](https://github.com/user-attachments/assets/b6762c16-130d-4e7b-ae25-91176565100e)

**Comment by isCopyman** [comment]
agree

**Comment by jackstine** [comment]
maybe a configuration color?

**Comment by orta** [comment]
I'd love user input to be blue

**Comment by webmemo-code** [comment]
> I'd love user input to be blue

I guess everybody has their preferred setting. It should be a configuration color ;-)

**Comment by tjx666** [comment]
@dicksontsai the use message color is same with the thinking text, too hard to distinct

<img width="602" height="465" alt="Image" src="https://github.com/user-attachments/assets/2c1b1dce-1318-49f2-8d11-04e17f2e7bbd" />

**Comment by webmemo-code** [comment]
Anthropic improved the layout a lot, well done! It's now much easier to identify your own contributions among Claude Code's output.

I'm closing the issue.

![Image](https://github.com/user-attachments/assets/b2c4ca05-d4d0-477b-a7c1-d5de05d9a4f3)
